blocks/explore-articles/explore-articles.js

```diff
@@ -153,79 +135,72 @@ const buildFirstArticles = (art, section) => {
-  const totalArticlesNumber = addAllArrays(groupedArticles);
+  const totalArts = addAllArrays(groupedArticles);
   const amountOfGroups = articleGroups.length;
 
-  const paginationSection = createElement('div', { classes: 'pagination-section' });
   const articlesSection = createElement('div', { classes: `${blockName}-articles` });
-
-  const amountOfArticles = createElement('p', { classes: 'article-amount' });
-  amountOfArticles.textContent = (totalArticlesNumber !== 0) ? `${totalArticlesNumber} articles` : getTextLabel('No article Message');
-
-  paginationSection.append(amountOfArticles);
-  articlesSection.append(paginationSection);
-
-  const moreSection = createElement('div', { classes: `${blockName}-more` });
-  const moreButton = createElement('button', { classes: 'more-btn' });
-  moreButton.textContent = getTextLabel('Load more articles button');
-  moreButton.addEventListener('click', (evt) => loadMoreArticles(evt, articleGroups, amountOfGroups));
-  if (totalArticlesNumber > articlesPerChunk) moreSection.append(moreButton);
+  const articlesContent = document.createRange().createContextualFragment(`
+      <div class="pagination-section">
+        <p class="article-amount">${totalArts !== 0 ? `${totalArts} articles` : getTextLabel('No article Message')}</p>
+      </div>
+      <div class="article-list"></div>
+      <div class="${blockName}-more">
+        ${totalArts > artsPerChunk ? `<button class="more-btn">${getTextLabel('Load more articles button')}</button>` : ''}
+      </div>
+    `);
 
   if (articleGroups.length !== 0) {
-    const articleListSection = createElement('div', { classes: 'article-list' });
-    buildFirstArticles(articleGroups, articleListSection);
-    articlesSection.append(articleListSection, moreSection);
+    buildFirstArticles(articleGroups, articlesContent.querySelector('.article-list'));
+    articlesContent.querySelector('.more-btn')?.addEventListener('click', (evt) => loadMoreArticles(evt, articleGroups, amountOfGroups));
   }
+  articlesSection.append(articlesContent);
+
   return articlesSection;
 };
 
 const handleForm = () => {
   counter = 1;
   const fieldset = document.querySelector('#explore-magazine-fieldset');
   const selects = fieldset.querySelectorAll('select');
-  const [category, truck] = selects;
-
-  const filteredList = allArticlesWithImage.filter((article) => {
-    const criteria = [
-      category.value === categoryPlaceholder && truck.value === truckPlaceholder,
-      category.value === categoryPlaceholder && truck.value === article.truck,
-      category.value === article.category && truck.value === truckPlaceholder,
-      category.value === article.category && truck.value === article.truck,
-    ];
-    const [criteria1, criteria2, criteria3, criteria4] = criteria;
-
-    if (criteria1 || criteria2 || criteria3 || criteria4) {
-      return article;
-    }
-    return null;
+  const [catSelect, truckSelect] = selects;
```

Comment:
we will have 3 filters, no only these 2, so ...

Reply:
yes, but not for this block